lib/qless/queue.rb

```diff
@@ -149,9 +160,9 @@ def stats(date = nil)
     # How many items in the queue?
     def length
       (@client.redis.multi do
-        @client.redis.zcard("ql:q:#{@name}-locks")
-        @client.redis.zcard("ql:q:#{@name}-work")
-        @client.redis.zcard("ql:q:#{@name}-scheduled")
+        %w[ locks work scheduled depends ].each do |suffix|
```

Comment:
`locks` corresponds to running jobs.  They are jobs that a worker currently holds a lock on.

Waiting jobs are in the `-work` sorted set (it is the set of jobs available to work on).

I agree that the sorted sets are confusing and a bit misnamed, but that ship sailed long ago...